md/release-notes.md

```diff
@@ -83,6 +83,10 @@ When defining actors in a pool, it is now possible to remove the "initiator" tag
 ### UI Designer properties autocomplete for variables
 Autocomplete capacity has been added in UI Designer properties fields, to help you input variable names with no risk of typos anymore.
 
+### UI Designer allows handling collections with invalid state 
+Collection with invalid content now keeps the model available allowing modification on the structure of the collect (for instance add and remove). 
+This is a behaviour change that allows forms to be better managed and further custom validations possible.
```

Comment:
I don't understand, sorry. 
Behaviour change? Do you mean I (as Fred) needs to do anything when I open my forms that use collections in a 7.6.1 UID or has it got no impact at all?
I don't get the value of "better management" and "further validations". Validation of what?